File: PATB/python/patb_1010.py
def delete(n):
    m = []
    i = 0
    tag = False
    while i + 1 < len(n) :
        if n[i+1] != 0:
            m.append(n[i]*n[i+1])
            m.append(n[i+1]-1)
            tag = True
        i += 2
    if not tag:
        m.append(0)
        m.append(0);
    return m

def main():
    n = [int(i) for i in input().split()]
    k = delete(n)
    print(" ".join(str(s) for s in k))

if __name__ == "__main__":
    main()

File: PATB/cpp/patb_1010.cpp
#include <iostream>
using namespace std;
int main() {
    int a, b;
    bool tag = false;
    while (cin >> a >> b) {
        if (b != 0) {
            if (tag == false) {
                cout << a * b << " " << b - 1;
            } else {
                cout << " " << a * b << " " << b - 1;
            }
            tag = true;
        }
    }

    if (tag == false) cout << "0 0" << endl;
    return 0;
}